heaps/min_heap.py

```diff
@@ -17,62 +17,97 @@ class MinHeap:
     def __init__(self):
         self.store = []
 
-
-    def add(self, key, value = None):
+    def add(self, key, value=None):
```

Comment:
👍  However the space complexity is O(log n) due to `heap_up`'s recursive stack trace.